tasks/elasticsearch-parameters.yml

```diff
@@ -47,3 +47,5 @@
 - set_fact: pid_dir={{ es_pid_dir }}/{{instance_suffix}}
 - set_fact: log_dir={{ es_log_dir }}/{{instance_suffix}}
 - set_fact: data_dirs={{ es_data_dirs | append_to_list('/'+instance_suffix) }}
+- set_fact: repo_dirs={{ es_repo_dirs | append_to_list('/'+instance_suffix) }}
```

Comment:
@remyma Is this a good idea to hase different repo name based on the `inventory_hostname` within the cluster?

I thought that all nodes have to mount the same filesystem to the same location on all master an data node ([more info](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html#_shared_file_system_repository))